js/templates/modals/editListing/editListing.html

```diff
@@ -65,21 +65,10 @@ <h1 class="h3 txUnb padMd js-listingHeading"><% print(ob.createMode ? ob.polyT('
                 <label for="editListingPrice" class="required"><%= ob.polyT('editListing.price') %></label>
                   <% if (ob.errors['listing.item.price']) print(ob.formErrorTmpl({ errors: ob.errors['listing.item.price'] })) %>
                 <div class="inputSelect">
-                  <%
-                    const decimalPlaces = ob.listing.metadata.pricingCurrency === 'BTC' ? 8 : 2;
-                    let listingPrice = ob.listing.item.price;
-                    listingPrice = typeof listingPrice === 'number' ? listingPrice.toFixed(decimalPlaces) : listingPrice;
-                  %>
-                  <input type="text" class="clrBr clrP js-price" name="listing.item.price" id="editListingPrice" value="<%= listingPrice%>" data-var-type="number">
+                  <input type="text" class="clrBr clrP js-price" name="listing.item.price" id="editListingPrice" value="<% if (typeof ob.listing.item.price === 'number') print(ob.formatPrice(ob.listing.item.price, ob.listing.metadata.pricingCurrency === 'BTC')) %>" data-var-type="number">
```

Comment:
To determine whether we want to add the decimal places in. If it's not set, it will be `undefined` and in that case, we don't want to output anything for the value attribute of this input (i.e. we want the field to be empty).

If it's a number, then we want to format it by adding the appropriate decimal places (e.g. 123 => 123.00).